WooCommerce/Classes/ViewRelated/Dashboard/Free Trial/FreeTrialSurvey/FreeTrialSurveyViewModel.swift

```diff
@@ -0,0 +1,82 @@
+import SwiftUI
+
+final class FreeTrialSurveyViewModel: ObservableObject {
```

Comment:
Nit: Similar comment as above about explaining the class.

Reply:
Done as part of https://github.com/woocommerce/woocommerce-ios/pull/10319/commits/fd5c8beda1db35ee4fd530d76fc6215399cc7b47